Doc changes related to default project source from Git repository (#1051)
codenvy/che-docs: src/main/pages/che-7/end-user-guide/proc_writing-a-devfile-for-your-project.adoc

  name: minimal-workspace
----

> Without any further configuration, a workspace with the default editor is launched along with its default plug-ins, which are configured on the {prod-short} Server. Che-Theia is configured as the default editor along with the *{prod-short} Machine Exec* plug-in. If workspace is launched within a git repository using factory, the project from given repository and branch will be created by default. Project name will match repository name.

Reviewer: ```suggestion
Without any further configuration, a workspace with the default editor is launched along with its default plug-ins, which are configured on the {prod-short} Server. Che-Theia is configured as the default editor along with the *{prod-short} Machine Exec* plug-in. When launching a workspace within a Git repository using a factory, the project from the given repository and branch is be created by default. The project name then matches the repository name.
```
